.github/workflows/build.yaml

```diff
@@ -82,3 +93,41 @@ jobs:
       with:
         name: Widelands ${{ matrix.config }} ${{ matrix.arch }} Installer
         path: ${{ github.workspace }}\Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.arch }}.exe
+
+  macos:
+    needs: [clang_tidy]
+    strategy:
+      matrix:
+        config:
+          - release
+          - debug
+        compiler:
+          - clang
+    name: MacOs ${{ matrix.config }} ${{ matrix.compiler }}
+    runs-on: macOS-latest
+    steps:
+    - name: Installing dependencies
+      run: |
+        export ICU_ROOT="$(brew --prefix icu4c)" && \
+        export PATH="/usr/local/opt/ccache/libexec:/usr/local/opt/gettext/bin:$PATH";
+        brew install boost ccache glew ninja sdl2 sdl2_image sdl2_mixer sdl2_ttf
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - name: Building
+      run: |
+        mkdir build_wl
+        cd build_wl/
+        ../utils/macos/build_app.sh ${{ matrix.compiler }} ${{ matrix.config }} ../
+        DMGPATH="$(pwd)"
+        DMGFILE="$(ls *.dmg)"
+        echo "DMG file is: $DMGPATH/$DMGFILE"
+        mv $DMGFILE ${{ github.workspace }}\Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.clang }}.dmg
```

Comment:
`matrix.clang` will *not* work ;) Neither will backslashes, probably

Reply:
Duh. Fixed.